package.json

```diff
@@ -8,8 +8,7 @@
     "test": "mocha \"test/**/*_test.js\"",
     "test-functional": "mocha \"test/functional/**/*_test.js\"",
     "test-mock-api": "mocha \"test/mock-api/**/*_test.js\"",
-    "coverage": "istanbul cover _mocha -- -u exports -R spec \"test/**/*_test.js\"",
-    "report-coverage": "cat ./coverage/lcov.info | coveralls"
```

Comment:
This is to run coverage on functional tests only (without running mock api). See: https://github.com/smartsheet/smartsheet-javascript-sdk/blob/mainline/ADVANCED.md#testing

Reply:
Was the original command incorrect then? Seems to me that

```
istanbul cover _mocha -- -u exports -R spec \"test/**/*_test.js\"
```

is running both the functional and mock api tests. If it was intending to only run the functional tests I would expect it to be

```
istanbul cover _mocha -- -u exports -R spec \"test/functional/**/*_test.js\"
```

Reply:
You're right, that would run both. Change looks good to me in that case.